content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -148,72 +137,52 @@ After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart butt
 
-.. image:: receipts_delivery_two_steps/two-step-sales-quote.png
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-button.png
    :align: center
-   :alt: After confirming the sales order, the Delivery smart button appears showing two items
-         associated with it.
+   :alt: Delivery smart button on sales order form.
 
 .. tip::
    Sales order receipts can also be found in the :menuselection:`Inventory` application. In the
    :guilabel:`Overview` dashboard, click the :guilabel:`# To Process` smart button in the
    :guilabel:`Pick` kanban card.
 
-   .. image:: receipts_delivery_two_steps/two-step-pick-kanban.png
-      :align: center
-      :alt: The pick order can be seen in the Inventory kanban view.
-
 Process the picking
 -------------------
 
-The picking and delivery order will be created once the sales order is confirmed. When the
-:guilabel:`Delivery` smart button appears, click it to reveal the :guilabel:`Transfers` dashboard,
-which lists both the picking and the delivery orders.
+Once the sales order is confirmed, a picking order is generated and ready to process.
 
-The status of the picking will be :guilabel:`Ready`, since the product must be picked from stock
-before it can be shipped. The status of the delivery order will be :guilabel:`Waiting Another
-Operation`, since the delivery cannot happen until the picking is completed. The status of the
-delivery order will only change to :guilabel:`Ready` once the picking has been marked as
-:guilabel:`Done`.
+Navigate to the :menuselection:`Inventory app`, and locate the :guilabel:`Pick` task card on the
+:guilabel:`Inventory Overview` dashboard. This reveals a list of all pickings to process.
 
-.. image:: receipts_delivery_two_steps/two-step-status.png
-   :align: center
-   :alt: Ready status for the pick operation while the delivery operation is Waiting Another
-         Operation.
-
-Click on the picking delivery order to begin processing it. If the product is in stock, Odoo will
-automatically reserve the product. Click :guilabel:`Validate` to mark the picking as
-:guilabel:`Done`, then the delivery order will be ready for processing. Since the documents are
-linked, the products which have been previously picked are automatically reserved on the delivery
+Click on the picking (WH/PICK) operation associated with the sales order to reveal the picking
 order.
 
-.. image:: receipts_delivery_two_steps/validate-two-step-pick.png
+Click :guilabel:`Check Availability` on the picking order form. If the product is in stock, Odoo
+automatically reserves the product. Quantities can also be manually set by changing the value in the
+:guilabel:`Quantity` column to match the value in the :guilabel:`Demand` column.
+
+Once ready, click :guilabel:`Validate` to complete the picking and move the product from
+:guilabel:`WH/Stock` to :guilabel:`WH/Output.`
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-picking-form.png
```

Comment:
Too zoomed in, would include the larger form/UI

Reply:
Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅